apps/spruce/src/components/CodeChanges/Diff.tsx

```diff
@@ -0,0 +1,60 @@
+import styled from "@emotion/styled";
+import { size } from "@evg-ui/lib/constants/tokens";
+
+export const Diff: React.FC<{ diff: string }> = ({ diff }) => {
+  const lines = diff.split("\n");
+  return (
+    <DiffContainer>
+      <Pre>
+        <Code>
+          {lines.map((line, i) => {
+            let Tag = BaseTag;
+            if (line.substring(0, 3) === "+++") {
+              Tag = Filestat;
+            } else if (line.substring(0, 3) === "---") {
+              Tag = Filestat;
+            } else if (line[0] === "+") {
+              Tag = Addition;
+            } else if (line[0] === "-") {
+              Tag = Deletion;
+            }
+
+            return (
+              // eslint-disable-next-line react/no-array-index-key
+              <Tag key={i}>
+                {line}
+                <br />
+              </Tag>
+            );
+          })}
+        </Code>
+      </Pre>
+    </DiffContainer>
+  );
+};
+
+const DiffContainer = styled.div`
+  padding: ${size.s};
+  width: 100%;
+`;
+
+const Pre = styled.pre`
+  max-width: 100%;
+  overflow-x: scroll;
+`;
+
+const Code = styled.code``;
+
+const BaseTag = styled.span``;
```

Comment:
Are these styles necessary?

Reply:
Yes for readability! But I removed `Code`